frontend/src/pages/Thema.css

```diff
@@ -0,0 +1,9 @@
+.nav nav-tabs{
+    display: flex;
+    flex-direction: row-reverse;
```

Comment:
hier sicher row-reverse als flex direction? eventuell kann man die Reihenfolge in den nav-tabs anpassen

Reply:
Dankeschön für die Hinweis! Ich probiere es später!